crates/primitives/src/transaction/mod.rs

```diff
@@ -46,6 +46,100 @@ pub(crate) static PARALLEL_SENDER_RECOVERY_THRESHOLD: Lazy<usize> =
         _ => 5,
     });
 
+/// Value is the type of the `value` field in the various Ethereum transactions structs.
+/// While the field is 256 bits, for many chains it's not possible for the field to use
+/// this full precision, hence we use a wrapper type to allow for overriding of encoding.
+#[derive_arbitrary(compact, rlp)]
+#[derive(Default, Debug, Copy, Clone, Hash, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Value(U256);
```

Comment:
I'm wondering if we should rename this to TransactionValue or TxValue?

Reply:
agree, renamed to TxValue (consistent with TxType)